File: GILua/src/main/java/org/anime_game_servers/gi_lua/models/scene/group/WorktopConfig.java
package org.anime_game_servers.gi_lua.models.scene.group;

import lombok.Getter;
import org.anime_game_servers.core.base.annotations.lua.LuaNames;

import java.util.Set;

@Getter
public class WorktopConfig {
    @LuaNames("init_options")
    private Set<Integer> initOptions;
    @LuaNames("is_persistent")
    private boolean isPersistent = false;
}

File: GILua/src/main/java/org/anime_game_servers/gi_lua/models/scene/group/WorktopConfig.kt
package org.anime_game_servers.gi_lua.models.scene.group

import org.anime_game_servers.core.base.annotations.lua.LuaNames

data class WorktopConfig (
    @field:LuaNames("init_options")
    val initOptions: Set<Int>? = null,
    @field:LuaNames("is_persistent")
    val isPersistent: Boolean = false
)
